scss/main.scss

```diff
@@ -67,8 +67,8 @@ $animation-length: 80s;
   background-color: none;
   transition: background-color 200ms linear;
   display: flex;
-  padding-top: 0.01rem;
-  padding-bottom: 0.01rem;
+  padding-top: 1%;
+  padding-bottom: 1%;
```

Comment:
i think fixed padding might be the move here or else the navbar can get too fat/skinny, maybe like 5px?